workflow/rules/common.smk

```diff
@@ -124,21 +124,35 @@ def get_map_reads_input_R2(wildcards):
     return ""
 
 
-def get_star_output(wildcards):
+def get_star_output(wildcards, fi="counts"):
```

Comment:
Maybe rename to `get_star_output_all_units()`, to distinguish more clearly from the new `get_star_bam()`, which is only for one unit, right?